src/kixi/gwyn/gwyn.clj

```diff
@@ -39,19 +41,56 @@
    (-> fire-station-lookup-table
        (wds/select-from-ds {:station fire-station})
        (wds/subset-ds :cols [:radius :lat :long]))})
+
+(def google-places-api "https://maps.googleapis.com/maps/api/place/nearbysearch/json?location=")
+
+(def api-key "&key=AIzaSyB4eWT7aEhsik-rP813fNJIN-qo12nhgIM")
```

Comment:
As mentioned, it's not a good idea to check API keys into GitHub as people can steal and misuse it. Definitely better to extract from config/use an environmental variable.